src/input_common/sdl/sdl_impl.cpp

```diff
@@ -1131,11 +1220,68 @@ ButtonMapping SDLState::GetButtonMappingForDevice(const Common::ParamPackage& pa
     return mapping;
 }
 
+ButtonMapping SDLState::GetDualControllerMapping(const std::shared_ptr<SDLJoystick>& joystick,
+                                                 const std::shared_ptr<SDLJoystick>& joystick2,
+                                                 ButtonBindings switch_to_sdl_button,
```

Comment:
```suggestion
                                                 const ButtonBindings& switch_to_sdl_button,
```

Ditto for ZButtonBindings below and also with the same parameters in GetSingleControllerMapping()